viya_ark_library/k8s/test_impl/response_data/resources_pgclusters.webinfdsvr.sas.com.json

```diff
@@ -0,0 +1,202 @@
+                "sas.com/default-database": "true"
+            },
+            "creationTimestamp": "2021-08-03T18:48:53Z",
+            "generation": 1,
+            "labels": {
+                "sas.com/admin": "cluster-local",
+                "sas.com/deployment": "sas-viya"
+            },
+            "managedFields": [
+                {
+                    "apiVersion": "webinfdsvr.sas.com/v1",
+                    "fieldsType": "FieldsV1",
+                    "fieldsV1": {
+                        "f:metadata": {
+                            "f:annotations": {
+                                ".": {},
+                                "f:kubectl.kubernetes.io/last-applied-configuration": {},
+                                "f:sas.com/default-database": {}
+                            },
+                            "f:labels": {
+                                ".": {},
+                                "f:sas.com/admin": {},
+                                "f:sas.com/deployment": {}
+                            },
+                            "f:ownerReferences": {}
+                        },
+                        "f:spec": {
+                            ".": {},
+                            "f:certframe": {
+                                ".": {},
+                                "f:containers": {},
+                                "f:initContainers": {},
+                                "f:volumes": {}
+                            },
+                            "f:connection": {
+                                ".": {},
+                                "f:host": {},
+                                "f:port": {},
+                                "f:ssl": {}
+                            },
+                            "f:database": {},
+                            "f:internal": {},
+                            "f:replicas": {},
+                            "f:rolesecret": {},
+                            "f:shutdown": {}
+                        }
+                    },
+                    "manager": "kubectl",
+                    "operation": "Update",
+                    "time": "2021-08-03T18:48:53Z"
+                }
+            ],
+            "name": "postgres",
+            "namespace": "d36363",
+            "ownerReferences": [
+                {
+                    "apiVersion": "orchestration.sas.com/v1alpha1",
+                    "blockOwnerDeletion": true,
+                    "controller": true,
+                    "kind": "SASDeployment",
+                    "name": "sas-viya",
+                    "uid": "74904415-de32-497f-9d5e-1b06c7058305"
+                }
+            ],
+            "resourceVersion": "287826",
+            "selfLink": "/apis/webinfdsvr.sas.com/v1/namespaces/d36363/pgclusters/postgres",
```

Comment:
Replace namespace value with "test" for consistency.

Reply:
updated.